api/bertyverifiablecreds.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package berty.account.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "berty.tech/berty/go/pkg/verifiablecredstypes";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+
+service VerificationCodeSender {
+  rpc SendVerificationCode (SendVerificationCode.Request) returns (SendVerificationCode.Reply);
+  rpc ValidateIdentifier (ValidateIdentifier.Request) returns (ValidateIdentifier.Reply);
+}
```

Comment:
This functionality being an integral part of the flow and not intended to be reused should perhaps be integrated as a function rather than split into a separate service.